practitioner-service/src/main/java/uk/ac/ox/ndph/mts/practitioner_service/validation/PractitionerValidation.java

```diff
@@ -46,6 +46,7 @@
     @Autowired
     public PractitionerValidation(PractitionerConfigurationProvider configurationProvider) {
         var configuration = configurationProvider.getConfiguration();
+        logger.info(Validations.CONFIGSERVER.message(), configuration);
```

Comment:
The enum doesn't include a placeholder for the configuration object. Does it work?

Reply:
@anatbal

Reply:
Redundant logging. Removed